modules/jtframe/bin/jtcore

```diff
@@ -523,61 +544,65 @@ if [ $SKIP_COMPILE = FALSE ]; then
         jtframe mmr $PRJ
         jtframe files syn $PRJ --target=$TARGET
         # Compilation
-        quartus_sh -v
-        quartus_sh --flow compile $corename > "$LOG"
-        if ! grep "Full Compilation was successful" "$LOG"; then
-            grep -i error "$LOG" -A 2
-            echo -e "${rBG}ERROR:${NC}${wBG} while compiling the project. ${red}Aborting${NC}"
-            report FAIL
-            exit 1
-        fi
-        if [ ! -e $corename.map.rpt ]; then
-            RPTPATH=$OUTPUTDIR
+        if [[ $TARGET = demist_zx3a* ]]; then
```

Comment:
we are indenting a lot of code for basically not doing anything in demist_zx3a, maybe a refactor here would help to make this more readable